lib/macho/exceptions.rb

```diff
@@ -127,8 +127,8 @@ class HeaderPadError < MachOError
     # @param filename [String] the filename
     def initialize(filename)
       super "Updated load commands do not fit in the header of " +
-      "#{filename}. #{filename} needs to be relinked, possibly with " +
-      "-headerpad or -headerpad_max_install_names"
+        "#{filename}. #{filename} needs to be relinked, possibly with " +
+        "-headerpad or -headerpad_max_install_names"
```

Comment:
RuboCop gives good advice here. Aside from the indentation fix, here's what you want to do:

```ruby
super "Updated load commands do not fit in the header of " \
  "#{filename}. #{filename} needs to be relinked, possibly with " \
  "-headerpad or -headerpad_max_install_names"
```

Concatenation of the strings with `String#+` will happen every time this code is executed, but if you're using line continuations (`\`) it will be the same as `"hello " "world"` (like in C) and the concatenation will already happen at parse time.